Update lora merging tutorials and update LoRA evaluation info (#532)
Lightning-AI/lit-gpt: tutorials/neurips_challenge_quickstart.md

(You can find a full task list in the task table [here](https://github.com/EleutherAI/lm-evaluation-harness/blob/master/docs/task_table.md).)

> To evaluate a LoRA-finetuned model, you need to first merge the LoRA weights with the base model to create a new checkpoint file:
> 
> 
> ```bash
> python scripts/merge_lora.py \
>   --checkpoint_dir "checkpoints/stabilityai/stablelm-base-alpha-3b/" \
>   --lora_path "out/stablelm3b/dolly/lora/experiment1/lit_model_lora_finetuned.pth" \
>   --out_dir "out/lora_merged/stablelm-base-alpha-3b/"
> ```
> 
> ```bash
> scp checkpoints/stabilityai/stablelm-base-alpha-3b/*.json \
> out/lora_merged/stablelm-base-alpha-3b/
> ```
> 
> For more information on LoRA weight merging, please see the
> [Merging LoRA Weights](https://github.com/Lightning-AI/lit-gpt/blob/main/tutorials/finetune_lora.md#merging-lora-weights)
> section of the LoRA finetuning documentation.

Reviewer: Should we just link to the lora weight merging section?

```suggestion
To evaluate a LoRA-finetuned model, you need to first merge the LoRA weights with the base model to create a new checkpoint file. Please follow the
[Merging LoRA Weights](https://github.com/Lightning-AI/lit-gpt/blob/main/tutorials/finetune_lora.md#merging-lora-weights)
section of the LoRA finetuning documentation.
```
